ui/src/app/pages/admin/admin-user-profile.tsx

```diff
@@ -394,12 +402,19 @@ const AccessModuleTable = (props: AccessModuleTableProps) => {
   );
 };
 
+const isCurrentUser = (userProfile: Profile): boolean => {
```

Comment:
What do you think `isAdminSelf` or `isSelf` for function name?

Reply:
How about isLoggedInUser?

Reply:
Okay.